File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/UDS/UDSDocumentUnitModel.cs
using System;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.UDS
{
    public class UDSDocumentUnitModel
    {
        public Guid UniqueId { get; set; }
        public UDSRelationType RelationType { get; set; }
        public Guid IdUDS { get; set; }
        public Guid IdDocumentUnit { get; set; }
        public int Environment { get; set; }
        public string RegistrationUser { get; set; }
        public DateTimeOffset RegistrationDate { get; set; }
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/UDS/UDSDocumentUnitModel.ts
import Relation = require('App/Models/UDS/UDSRelationModel');

class UDSDocumentUnitModel {
    constructor() {
    }

    IdUDS: string;
    Relation: Relation;
    UniqueId: string;
    SourceUDS: Relation;
}

export = UDSDocumentUnitModel;